addOns/ascanrules/CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Fixed
+- SQL rule should not target NoSQL Dbs.
```

Comment:
Should this be done in all the SQLi rules?

Reply:
Not according to https://www.zaproxy.org/techtags/db.mongodb/ 😁

Reply:
Okay